mod/wikirate/set/right/contributed_metrics.rb

```diff
@@ -1,19 +1,20 @@
+def contribution_count
+  @cc ||= Card.search :type_id=>MetricID, :or=>
+                      {
+                       :left=>cardname.left,
```

Comment:
It is supposed to find every metric designed by "left". I don't see why it won't do that.